### PR TITLE
fix writing anonymized empty room data

### PR DESCRIPTION
**Pull request by agramfort**
cc @hoechenberger @jasmainak

### CODE REVIEW

File: doc/whats_new.rst [hunk 1]
```diff
@@ -39,6 +39,7 @@ Enhancements
 Bug fixes
 ^^^^^^^^^
 - The function :func:`mne_bids.write_raw_bids` now outputs ``*_electrodes.tsv`` and ``*_coordsystem.json`` files for EEG/iEEG data that are BIDS-compliant (only contain subject, session, acquisition, and space entities), by `Adam Li`_ (`#601 <https://github.com/mne-tools/mne-bids/pull/601>`_)
+- Make sure writing empty room data with anonymization shifts the session back in time, by `Alexandre Gramfort`_ (`#611 <https://github.com/mne-tools/mne-bids/pull/611>`_)
```
Comment (hoechenberger) [inline doc/whats_new.rst:42]:
```suggestion
- Make sure writing empty-room data with anonymization shifts the session back in time, by `Alexandre Gramfort`_ (`#611 <https://github.com/mne-tools/mne-bids/pull/611>`_)
```

**Review by hoechenberger — APPROVED** [review]
LGTM if CI goes green!!

**Comment by hoechenberger** [comment]
Thanks @agramfort

**Comment by jasmainak** [comment]
Thanks @agramfort !

**Pull request merged**